pkg/ddevapp/config.go

```diff
@@ -1382,6 +1381,26 @@ func WriteImageDockerfile(fullpath string, contents []byte) error {
 	return nil
 }
 
+// IsAllowedProjectLocation returns an error if the project location is not allowed
+func IsAllowedProjectLocation(app *DdevApp) error {
+	homeDir, _ := os.UserHomeDir()
+	if app.AppRoot == filepath.Dir(globalconfig.GetGlobalDdevDir()) || app.AppRoot == homeDir {
+		return fmt.Errorf("'ddev config' is not useful in your home directory (%v)", homeDir)
+	}
+	projectList := globalconfig.GetGlobalProjectList()
+	for _, project := range projectList {
```

Comment:
This is a good and adequate strategy, although I have accidentally created projects that have a .ddev in a subdirectory. It's typically when I have done something like a `ddev stop --unlist -a`, removing everything from global project list, or when my global config has been damaged somehow.